teams/team-cloudbees.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "cloudbees-developers"
```

Comment:
Would it not make sense to use the same name as the file? (Also should the file start with "team-" as it is in the teams folder and team is obvious from the `@name` notation?

Reply:
Yeah. I don't disagree actually. The filename should `xyz.yml` as it is currently.
I thought about dropping the `name` attribute from the team yaml, and using the filename as the team name too. 
The reason why I didn't go that path (yet) is that I was unsure we'd not want a team name with characters that wouldn't fit well a filename.

Not fully decided yet. I'll _literally_ sleep on it :-).

What about the rest?

Reply:
better to have the name in the yaml file imo, more obvious and flexible I think.

I agree no need to have team in the file name, but not a blocker

Reply:
> better to have the name in the yaml file imo, more obvious

I would say it is more obvious when you have the file open - but less obvious if you want to have a look for and edit a file for a given team (as there is nothing ensuring they match).

hopefully there won't be too many of these files(?) so having a different team name from the file name will not be an issue.

Reply:
I suppose I can add a check to enforce the `name` of the filename do match, at least for now. I don't like the repetition, but well I can live with it.

Reply:
done